icons/edit-2.svg

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why the blank line?

Reply:
Was editing it to try to make the SVG appear in profile.html and must have missed this.

Reply:
I see, all good. We'll catch that kind of thing in a cleanup pass later.